apps/explorer/webpack.config.js

```diff
@@ -11,6 +12,12 @@ module.exports = (config, context) => {
     : [];
   return {
     ...config,
-    plugins: [...additionalPlugins, ...config.plugins],
+    plugins: [
+      ...additionalPlugins,
+      ...config.plugins,
+      new webpack.DefinePlugin({
+        'process.platform': JSON.stringify(process.platform),
```

Comment:
not sure about this part. turned out it's needed by `better-ajv-errors` (https://github.com/atlassian/better-ajv-errors). 

i wasn't aware that it's not designed to be used in the browser (see: https://github.com/atlassian/better-ajv-errors/issues/40). it improves the way errors are read, and as it stands, this hack makes it work in the browsers, too, but I'm not sure if this is worth doing here. let me know your thoughts.

Reply:
I hadn't realised ajv was not suitable browsers. As discussed on slack, can we take a look at using zod?

Reply:
@mattrussell36 thanks for the suggestion! zod looks pretty neat, had to make some adjustments with the error handling, but refactored my PR so that is uses zod now for validation.